### PR TITLE
chore: Add PR note to all templates

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

Update issue templates

#### What changes did you make? (Give an overview)

Included the note to not open a PR until the issue has been accepted on all issue templates.

### CODE REVIEW

**Comment by amareshsm** [comment]
Should we add this note in other repos too?

**Review by amareshsm — APPROVED** [review]
LGTM. Waiting for 2nd approval.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
